src/commands/mapsheet-coverage/mapsheet.coverage.ts

```diff
@@ -42,6 +42,39 @@ function forceMultiPolygon(f: GeoJSON.Feature): GeoJSON.Feature<GeoJSON.MultiPol
   throw new Error(`${f.geometry.type}: is not a polygon`);
 }
 
+/**
+ * a polygon of 0.00001 x 0.00001 (about 1m x 1m) converts to 1e-10 degrees^2
```

Comment:
Looks correct based on https://earthscience.stackexchange.com/a/6846/11352 and a test of https://www.stevemorse.org/nearest/distance.php using the coordinates of Wellington.